### PR TITLE
Updates knife_windows docs

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

Errs on the side of over-completion.

Updates the:
- [x] bootstrap_windows_ssh 
- [x] bootstrap_windows_winrm
- [x] cert generate
- [x] cert install
- [x] listener create
- [x] winrm

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by tas50** [comment]
Looks like we were missing a few things there...

File: chef_master/source/knife_windows.rst [hunk 1]
```diff
@@ -163,59 +163,104 @@ This argument has the following syntax:
...
 
-``--auth-timeout MINUTES``,
-   The amount of time (in minutes) to wait for authentication to succeed. Default: ``2``.
+``-A``, ``--forward-agent``
+    Enables SSH agent forwarding.
 
-``--bootstrap-no-proxy NO_PROXY_URL_or_IP``
-   A URL or IP address that specifies a location that should not be proxied.
+``--auth-timeout MINUTES``
+    The maximum time in minutes to wait to for authentication over the transport to the node to succeed. Default: ``2``.
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: `to wait to for` -> `to wait for`

File: chef_master/source/knife_windows.rst [hunk 2]
```diff
@@ -163,59 +163,104 @@ This argument has the following syntax:
...
+    The SSH tunnel or gateway that is used to run a bootstrap action on a machine that is not accessible from the workstation.
 
-``-i IDENTITY_FILE``, ``--identity-file IDENTITY_FILE``
-   The SSH identity file used for authentication. Key-based authentication is recommended.
+``--hint HINT_NAME[=HINT_FILE]``
+    Specify Ohai Hint to be set on the bootstrap targeting multiple nodes. See ``--hint options`` to specify multiple hints.
 
-``-j JSON_ATTRIBS``, ``--json-attributes JSON_ATTRIBS``
-   A JSON string that is added to the first run of a chef-client.
+``--[no]host-key-verify``
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: should this be `[no-]` with a tailing dash?

File: chef_master/source/knife_windows.rst [hunk 3]
```diff
@@ -163,59 +163,104 @@ This argument has the following syntax:
...
+    Use to disable host key verification. Default: ``true``.
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: This description assumes that `no-` was specified and it should probably read `Use to enable`

File: chef_master/source/knife_windows.rst [hunk 4]
```diff
@@ -163,59 +163,104 @@ This argument has the following syntax:
...
-   The name of the node.
+    The name of the node.
+
+``--node-ssl-verify-mode [peer|none]``
+    Whether or not to verify the SSL cert for all requests.
 
-``--[no-]host-key-verify``
-   Use ``--no-host-key-verify`` to disable host key verification. Default setting: ``--host-key-verify``.
+``--[no-]node-verify-api-cert``
+    Verify the SSL cert for HTTPS requests to the Chef sAPI. Default: ``true``.
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: I think this is `chef server` instead of `chef api`

File: chef_master/source/knife_windows.rst [hunk 5]
```diff
@@ -251,39 +296,135 @@ Options
...
    Indicates the client should be installed as a Windows Service.
 
 ``-j JSON_ATTRIBS``, ``--json-attributes JSON_ATTRIBS``
    A JSON string that is added to the first run of a chef-client.
 
+``--json-attribute-file FILE``
+    A JSON file used at the first chef-client run.
+
+``-m``, ``--manual-list``
+    QUERY is a space-separated list of servers.
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: Just `A space-separated list of servers`

**Review by mwrock — CHANGES_REQUESTED** [review]
This is MUCH BETTER!

File: chef_master/source/knife_windows.rst [hunk 6]
```diff
@@ -251,39 +296,135 @@ Options
...
+    The codepage to use for the winrm cmd shell.
+
+``--winrm-shell SHELL``
+      The WinRM shell type. Values: ``cmd``, ``powershell``, ``elevated``. ``elevated`` runs powershell in a scheduled task
+
+``--winrm-ssl-verify-mode SSL_VERIFY_MODE``
+    The WinRM peer verification mode. Values: ``verify_peer``, ``verify_none``
+
+``-x USERNAME``, ``--ssh-user USERNAME``
+    The SSH username. Default: ``root``.
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: The WinRM username. Default: `Administrator`

File: chef_master/source/knife_windows.rst [hunk 7]
```diff
@@ -251,39 +296,135 @@ Options
...
+``--winrm-codepage CODEPAGE``
+    The codepage to use for the winrm cmd shell.
+
+``--winrm-shell SHELL``
+      The WinRM shell type. Values: ``cmd``, ``powershell``, ``elevated``. ``elevated`` runs powershell in a scheduled task
+
+``--winrm-ssl-verify-mode SSL_VERIFY_MODE``
+    The WinRM peer verification mode. Values: ``verify_peer``, ``verify_none``
+
+``-x USERNAME``, ``--ssh-user USERNAME``
```
Comment (mwrock) [inline chef_master/source/knife_windows.rst]: `--winrm-user`

**Review by mwrock — APPROVED** [review]

**Pull request merged**